examples/tutorials/nb_python/managed_rigid_object_tutorial.py

```diff
@@ -0,0 +1,560 @@
+
+    observations += simulate(sim, dt=1.0, get_frames=True)
+
+    if make_video:
+        vut.make_video(
+            observations,
+            "rgba_camera_1stperson",
+            "color",
+            output_path + "velocity_control",
+            open_vid=show_video,
+        )
+
+    # [/velocity_control]
+    # %%
+    # [local_velocity_control]
+
+    vel_control.linear_velocity = np.array([0, 0, 2.3])
+    vel_control.angular_velocity = np.array([-4.3, 0.0, 0])
+    vel_control.lin_vel_is_local = True
+    vel_control.ang_vel_is_local = True
+
+    observations = simulate(sim, dt=1.5, get_frames=True)
+
+    # video rendering
+    if make_video:
+        vut.make_video(
+            observations,
+            "rgba_camera_1stperson",
+            "color",
+            output_path + "local_velocity_control",
+            open_vid=show_video,
+        )
+
+    # [/local_velocity_control]
+    rigid_obj_mgr.remove_all_objects()
+    # %%
+    # [embodied_agent]
+
+    # load the lobot_merged asset
+    locobot_template_id = obj_templates_mgr.load_configs(
+        str(os.path.join(data_path, "objects/locobot_merged"))
+    )[0]
+
+    # add robot object to the scene with the agent/camera SceneNode attached
+    locobot = rigid_obj_mgr.add_object_by_id(
+        locobot_template_id, sim.agents[0].scene_node
+    )
+    locobot.translation = np.array([1.75, -1.02, 0.4])
+
+    vel_control = locobot.velocity_control
+    vel_control.linear_velocity = np.array([0, 0, -1.0])
+    vel_control.angular_velocity = np.array([0.0, 2.0, 0])
+
+    # simulate robot dropping into place
+    observations = simulate(sim, dt=1.5, get_frames=make_video)
+
+    vel_control.controlling_lin_vel = True
+    vel_control.controlling_ang_vel = True
+    vel_control.lin_vel_is_local = True
+    vel_control.ang_vel_is_local = True
+
+    # simulate forward and turn
+    observations += simulate(sim, dt=1.0, get_frames=make_video)
+
+    vel_control.controlling_lin_vel = False
+    vel_control.angular_velocity = np.array([0.0, 1.0, 0])
+
+    # simulate turn only
+    observations += simulate(sim, dt=1.5, get_frames=make_video)
+
+    vel_control.angular_velocity = np.array([0.0, 0.0, 0])
+    vel_control.controlling_lin_vel = True
+    vel_control.controlling_ang_vel = True
+
+    # simulate forward only with damped angular velocity (reset angular velocity to 0 after each step)
+    observations += simulate(sim, dt=1.0, get_frames=make_video)
+
+    vel_control.angular_velocity = np.array([0.0, -1.25, 0])
+
+    # simulate forward and turn
+    observations += simulate(sim, dt=2.0, get_frames=make_video)
+
+    vel_control.controlling_ang_vel = False
+    vel_control.controlling_lin_vel = False
+
+    # simulate settling
+    observations += simulate(sim, dt=3.0, get_frames=make_video)
+
+    # remove the agent's body while preserving the SceneNode
+    rigid_obj_mgr.remove_object_by_ID(locobot.ID, delete_object_node=False)
```

Comment:
I assume you are not deleting the object node as otherwise that would delete the node for the agent?  If yes, I think we should have `add_object_by_id` take the parent node, not the node, as otherwise this seems like it'll be a frequent gotcha

Reply:
Agreed it is ugly and leads to easy pitfalls. There is a minor complication here. Physics will update the object's node, so if we create a child internally and attach a RigidObject feature to the child, the agent node with the sensors won't be affected by changes to the object such as dynamics.